test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/StringHtmlContentTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Framework.WebEncoders.Testing;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Rendering
+{
+    public class StringHtmlContentTest
+    {
+        [Fact]
+        public void CreateStringHtmlContent()
+        {
+            // Arrange & Act
+            var content = new StringHtmlContent("Hello World");
+
+            // Assert
+            var result = Assert.IsType<StringHtmlContent>(content);
```

Comment:
What is this testing? You just instantiated the `StringHtmlContent` instance so of course that's its type.